Trappist/src/Promact.Trappist.Core/Controllers/CategoryController.cs

```diff
@@ -26,44 +27,47 @@ public CategoryController(ICategoryRepository categoryRepository)
 
         [HttpPost]
         /// <summary>
-        /// Post Method 
-        /// Will Add a Catagory Name in Category Model
+        /// Method to Add Category
```

Comment:
It is not a method. It is an API. so description should be API to add a category.